javascript-light-bulb/main.js

```diff
@@ -0,0 +1,16 @@
+var $lightBulb = document.querySelector('.bulb');
+var $lightStatus = document.querySelector('.off');
+var clicks = 0;
+
+function clicksFunction(event) {
+  clicks++;
+  if (clicks % 2 === 0) {
```

Comment:
To determine whether to turn the light on or off, you are making a counter, but the only thing you ever care about is if the counter's value was even or odd. This is a situation where you would be better off using a boolean to keep track of if the light is currently on.